src/Admin/SupportLink.php

```diff
@@ -112,8 +110,8 @@ public function amp_validated_url_status_actions( $actions, WP_Post $post ) {
 		}
 
 		$query_args = [
-			'page'    => 'amp-support',
-			'post_id' => $post->ID,
+			'page' => 'amp-support',
+			'url'  => rawurlencode( $post->post_title ),
```

Comment:
It would be preferable to use `\AMP_Validated_URL_Post_Type::get_url_from_post()` to obtain the URL from a validated URL post:

```suggestion
			'url'  => rawurlencode( AMP_Validated_URL_Post_Type::get_url_from_post( $post ) ),
```